server/log_rpc_server_test.go

```diff
@@ -553,16 +553,17 @@ func TestGetLatestSignedLogRoot2(t *testing.T) {
 			noCommit: true,
 		},
 		{
-			// Test error case where storage fails to commit the tx.
-			req:       getLogRootRequest1,
-			errStr:    "commit",
-			commitErr: errors.New("commit() error"),
+			// Test error case where the log root could not be read
```

Comment:
This looks like a new test case. Please preserve the old "commit" test case.

Reply:
This is a new test case. The old "commit" test case is no longer used because of the timeline change of TX commit in "GetLatestSignedLogRoot()". Now since we will try to get the consistency proof, the TX commit is moved to the end.

Reply:
I see. Thanks